### DISCUSSION TITLE
Supabase won't allow me to "UPDATE" - Success. No rows returned

### DISCUSSION THREAD

**Discussion by alexjs-dev** [supabase/supabase#20215] [Questions]
Edit: Solved. Thanks!

**Comment by GaryAustin1** [comment]
Your terminology is somewhat confusing as you say GET and CREATE but show INSERT AND UPDATE which are SQL methods.

You appear to be running SQL so you would simply google Posgtres return data from update.
This is first hit: https://www.postgresql.org/docs/current/dml-returning.html
